docs/coding-guidelines/breaking-change-rules.md

```diff
@@ -236,6 +236,8 @@ successfully bind to that overload, if simply passing an `int` value. However, i
 
     Definite assignment rules allow use of uninitialized variables so long as the variable type is a stateless struct. If the struct is made stateful, code could now end up with uninitialized data. This is both potentially a source breaking and binary breaking change.
 
+* Adding a `ref` (byref), object reference, or generic type parameter-based field to a value type that formerly had none of those field kinds. If the value type already contains at least one such field, adding another is non-breaking.
```

Comment:
```suggestion
* Adding a `ref` (byref), object reference, or generic type that does not have `unmanaged` constrain parameter-based field to a value type that formerly had none of those field kinds. If the value type already contains at least one such field, adding another is non-breaking.
```
If we want to be precise, adding a new field of a generic type that is constrained as `unmanaged` should be fine.